pkg/kubeapiserver/default_storage_factory_builder.go

```diff
@@ -24,9 +24,8 @@ import (
 	"k8s.io/apiserver/pkg/storage/storagebackend"
 )
 
-// SpecialDefaultResourcePrefixes are prefixes compiled into Kubernetes.
```

Comment:
what's the reason for unexporting this?

Reply:
Not used by outer, so i think it can be set unexported

Reply:
> Not used by outer, so i think it can be set unexported

It is used by other packages, just not in this repo

Reply:
ok, will revert this.

Reply:
done